### ISSUE TITLE
Error with trakt.tv script

### ISSUE THREAD

**Issue by thwaller** [tknorris/1Channel#15]
See here:
https://github.com/trakt/script.trakt/issues/304

Moved here per tknorris request.

**Comment by tknorris** [comment]
A full log of the error with debug turned on would help

**Comment by thwaller** [comment]
Ok, I will watch an episode, but it a 60min. Any way to speed it up for this purpose?

**Comment by tknorris** [comment]
I only need the log until the message pops. It would also help to turn on add-on debug in the trakt addon

**Comment by thwaller** [comment]
[log.txt](https://github.com/tknorris/1Channel/files/243834/log.txt)

Here is a log. Delete when done, I do not want to delete it before you are done with it.

**Comment by tknorris** [comment]
So, the problem is that the show title lookup on trakt fails from 1CH fails using the show title returned by metahandler. There's no real fix to be done. If the titles don't match then the scrobble won't work. We have an override that we can use, but it won't work in this case because we don't have any of the reference ids like tvdb, tmdb, or imdb.

**Comment by tknorris** [comment]
To be more specific, 1CH has the show as "House of Cards (US)" while trakt has it as "House of Cards", so the scrobble by title fails, thus the warning.

**Comment by thwaller** [comment]
Reference IDs? Pages exist on TMDB, TVDB, etc for this. Is that what you mean?

**Comment by tknorris** [comment]
Yeah, but 1ch doesn't know about them

**Comment by tknorris** [comment]
It only pulls imdb from primewire if it can

**Comment by thwaller** [comment]
So, I can go there to add data?

**Comment by thwaller** [comment]
https://www.primewire.ag/watch-2759743-House-of-Cards-online-free
links up to 
http://www.imdb.com/title/tt1856010/

**Comment by tknorris** [comment]
No it just won't work period

**Comment by thwaller** [comment]
Ok. One last question as I am trying to understand. Where exactly is the name of "House of Cards (US)" getting introduced? Specifically the "(US)" portion. I am trying to see where this is caused for future reference as this is not an isolated problem to this series.

**Comment by tknorris** [comment]
If have to go back thru the code but I think it comes from metahandler.

**Comment by thwaller** [comment]
If it helps anyone, there is a work around, or fix, for this that I have found and tested to work. WIth the creation of nfo files for my library items, I am able to change the name of the movie and/or show. In this example, I was able to remove the "(US)" from the name. With that, scrobbling, watched flag and prompting for rating has all returned for this specific tv show. I have tested this on others that had strange issues and all seem to be resolved as it would relate to naming differences.

> To be more specific, 1CH has the show as "House of Cards (US)" while trakt has it as "House of Cards", so the scrobble by title fails, thus the warning.
The above corrects this completely, All now see the show as "House of Cards" as the nfo translates the "(US)" out.
